### PR TITLE
chore: upgrade flutter and update dependencies

### PR DESCRIPTION
**Pull request by Xazin**
Closes: #198

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -21,59 +21,59 @@ environment:
...
   mocktail: ^0.2.0
-  package_info_plus: ^1.0.6
-  phone_number: ^0.11.0+2
-  rive: 0.7.28
-  rxdart: ^0.27.1
-  share_plus: ^2.1.4
-  shared_preferences: ^2.0.7
+  package_info_plus: ^1.3.0
+  phone_number: ^0.12.0+1
+  rive: 0.7.28 # Why is this a specific version?
```
Comment (Xazin) [inline pubspec.yaml:57]: I still have this question.. 😂

Comment (wizlif) [reply pubspec.yaml:57]: Tests seemed to fail if we used any above that. Not sure if it was fixed in the latest versions

**Review by wizlif — APPROVED** [review]

**Comment by Xazin** [comment]
I guess the reason why tests once failed due to specific Rive version had something to do with us testing screens and not widgets, so there was no way to optimize this.

Since we no longer tests screen we won't have this problem in the future.

**Comment by wizlif** [comment]
> I guess the reason why tests once failed due to specific Rive version had something to do with us testing screens and not widgets, so there was no way to optimize this.
> 
> Since we no longer tests screen we won't have this problem in the future.

True, guess we dodged a bullet there

**Review by wizlif — APPROVED** [review]

**Pull request merged**